images/custom-error-pages/main.go

```diff
@@ -146,6 +179,8 @@ func errorHandler(path string) func(http.ResponseWriter, *http.Request) {
 		log.Printf("serving custom error response for code %v and format %v from file %v", code, format, file)
 		io.Copy(w, f)
 
+		w.WriteHeader(code)
```

Comment:
Will not work, because in previous line data from file `f` was copied to `w` and now `WriteHeader` just do nothing;
I suggest moving it to an original position, around ~136 line -> just after `code` was able to be changed last time.

Reply:
fixed. Thanks for suggestion!